client/src/Views/Home/Campaigns/Campaign/Step.js

```diff
@@ -45,6 +50,37 @@ const EmailTooltip = withStyles({
 
 const Step = props => {
   const classes = useStyles();
+  const [anchorEl, setAnchorEl] = React.useState(null);
+
+  const openMenu = e => {
+    setAnchorEl(e.currentTarget);
+  };
+
+  const closeMenu = () => {
+    setAnchorEl(null);
+  };
+
+  const moveProspects = async () => {
+    try {
+      const prospectIds = [];
+
+      props.prevStep.prospects.forEach(prevProspect => {
+        //Check if prospect already exists in current step
+        const duplicate = props.step.prospects.some(curr => {
+          return curr.prospectId === prevProspect.prospectId;
+        });
+        if (!duplicate) {
+          prospectIds.push(prevProspect.prospectId);
+        }
+      });
```

Comment:
Instead of doing this logic on the front-end, we'd probably want to do it on the back-end (meaning creating another API that lets us specify that we want to move all prospects from previous step), as it's not feasible to make sure we have access to all the prospects on the front-end when the number of prospects in a specific campaign grows. However, this is fine for now